### PR TITLE
docs: updated tls version support for iOS

### PR DESCRIPTION
**Pull request by vijaysingh-axway**
https://jira.appcelerator.org/browse/TIMOB-26959

### CODE REVIEW

File: apidoc/Titanium/Network/HTTPClient.yml [hunk 1]
```diff
@@ -97,14 +97,52 @@ description: |
     In Titanium, if a connection with TLS 1.2 fails, Titanium will re-attempt the connection with
     TLS 1.1 and TLS 1.0. By default, TLS 1.2 is attempted first.
 
-    Use the <Titanium.Network.HTTPClient.tlsVersion> property to set the version of the TLS protocol
+    In android, use the <Titanium.Network.HTTPClient.tlsVersion> property to set the version of the TLS protocol
```
Comment (janvennemann) [inline apidoc/Titanium/Network/HTTPClient.yml:100]:
```suggestion
    On Android, use the <Titanium.Network.HTTPClient.tlsVersion> property to set the version of the TLS protocol
```

File: apidoc/Titanium/Network/HTTPClient.yml [hunk 2]
```diff
@@ -97,14 +97,52 @@ description: |
...
     if you know the version the server is running.  If you do not know, do not set this property.
-    Titanium will not fallback with a lower TLS version if the `tlsVersion` property is set.
+
+    In iOS, add `NSExceptionMinimumTLSVersion` key in plist section of tiapp.xml to set minimum TLS version.
```
Comment (janvennemann) [inline apidoc/Titanium/Network/HTTPClient.yml:103]:
```suggestion
    On iOS, add the `NSExceptionMinimumTLSVersion` key in the `<plist>` section of `tiapp.xml` to set a minimum TLS version. 
```

File: apidoc/Titanium/Network/HTTPClient.yml [hunk 3]
```diff
@@ -97,14 +97,52 @@ description: |
     In Titanium, if a connection with TLS 1.2 fails, Titanium will re-attempt the connection with
     TLS 1.1 and TLS 1.0. By default, TLS 1.2 is attempted first.
 
-    Use the <Titanium.Network.HTTPClient.tlsVersion> property to set the version of the TLS protocol
+    In android, use the <Titanium.Network.HTTPClient.tlsVersion> property to set the version of the TLS protocol
     if you know the version the server is running.  If you do not know, do not set this property.
-    Titanium will not fallback with a lower TLS version if the `tlsVersion` property is set.
+
+    In iOS, add `NSExceptionMinimumTLSVersion` key in plist section of tiapp.xml to set minimum TLS version. 
+    See following example-
```
Comment (janvennemann) [inline apidoc/Titanium/Network/HTTPClient.yml:104]:
```suggestion
    See following example:
```

The examples also feels kind of redundant. I think only the last one should be enough. Also make sure to use proper indentation and fenced code blocks.

```
    ```
    <key>NSAppTransportSecurity</key>
    <dict>
      <key>NSExceptionDomains</key>
      <dict>
        <key>www.appcelerator.com</key>
        <dict>
          <key>NSExceptionMinimumTLSVersion</key>
          <string>TLSv1.3</string>
        </dict>
      </dict>
    </dict>
    ```
```

File: apidoc/Titanium/Network/HTTPClient.yml [hunk 4]
```diff
@@ -539,15 +577,15 @@ properties:
     description: |
         If you experience handshake failures, set this value to a lower version using the TLS
         constants in <Titanium.Network>. 'undefined', 'null', or unsupported values use the default
-        behavior for that iOS and Android versions.
+        behavior for that android versions.
```
Comment (janvennemann) [inline apidoc/Titanium/Network/HTTPClient.yml:580]:
```suggestion
        behavior for that Android versions.
```

**Review by janvennemann — CHANGES_REQUESTED** [review]

**Comment by vijaysingh-axway** [comment]
@janvennemann  Addressed review comments.

**Review by janvennemann — APPROVED** [review]

**Comment by ssekhri** [comment]
FR Passed.

**Pull request merged**